index.php

```diff
@@ -81,7 +81,7 @@
 				</div>
 				
 				<div class = "headertext2">
-					<a href="index.php#search_content"><h2>Browse Now</h2></a>
+					<form action="index.php#search_content"><input style = "font-size: 2.5vw; width: 21vw; height: 3.7vw; margin: 0vw 0vw 0vw 0vw;" type = "submit" value = "Browse Now"/></form>

```

Comment:
Added in button which scrolls the page down to the main content area (search, genre covers, etc...)